packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -29,6 +31,9 @@ abstract class ScrollActivityDelegate {
   /// The direction in which the scroll view scrolls.
   AxisDirection get axisDirection;
 
+  /// The current scrollPosition in which the scroll view scrolls.
+  ScrollPosition get scrollPosition;
```

Comment:
The scroll activity delegate is implemented by a scroll position, why is this needed?